src/main/java/org/folio/orders/utils/ErrorCodes.java

```diff
@@ -47,7 +47,9 @@
   MISSING_CONTRIBUTOR_NAME_TYPE("missingContributorNameType", "The specified contributor-name-types were not found in inventory-storage"),
   MISSING_LOAN_TYPE("missingLoanType", "Loan-type is a required field for creating an Item in Inventory"),
   ENCUMBRANCE_CREATION_FAILURE("encumbranceCreationFailure", "One or more encumbrance record(s) failed to be created"),
-  PROHIBITED_FIELD_CHANGING("protectedFieldChanging","Protected fields can't be modified");
+  PROHIBITED_FIELD_CHANGING("protectedFieldChanging","Protected fields can't be modified"),
+  ORDER_UNITS_NOT_FOUND("orderUnitsNotFound", "Order doesn't contain unit ids or units not found"),
+  USER_HAS_NOT_PERMISSIONS("userHasNoPermission", "User hasn't permissions - operation is restricted");
```

Comment:
Looks like a typo: Must be USER_HAS_**NO**_PERMISSIONS
Also the description should be, User has no permissions

Reply:
Fixed.